pkg/ebpf/c/tracee.bpf.c

```diff
@@ -664,11 +679,23 @@ BPF_HASH(new_module_map, u64, kernel_new_mod_t, MAX_NUM_MODULES);
 // a map), but an attacker might start unloading modules in the background and
 // race with the check in order to abort reporting for hidden modules.
 
-BPF_LRU_HASH(recent_deleted_module_map, u64, kernel_deleted_mod_t, 50);
-BPF_LRU_HASH(recent_inserted_module_map,
-             u64,
-             kernel_new_mod_t,
-             50); // Likewise for module insertion
+struct recent_deleted_module_map {
+    __uint(type, BPF_MAP_TYPE_LRU_HASH);
+    __uint(max_entries, 50);
+    __type(key, u64);
+    __type(value, kernel_deleted_mod_t);
+} recent_deleted_module_map SEC(".maps");
+
+typedef struct recent_deleted_module_map recent_deleted_module_map_t;
```

Comment:
Are you typedefing these because it will be versioned and we will need to give compiler hints or for consistency's sake?

Reply:
The initial idea emerged from that case (compiler hints) but as the removal of macros was already intended the typedef are to let all type definitions in the same pattern of the types.h ones.